src/_nebari/upgrade.py

```diff
@@ -1099,16 +1159,15 @@ def _version_specific_upgrade(
                 rich.print(
                     "[red bold]Before upgrading, you need to manually delete the prometheus-node-exporter daemonset and update the kube-prometheus-stack CRDs. To do that, please run the following commands.[/red bold]"
                 )
-                Prompt.ask("Hit enter to show the commands")
+                _ = attempt_fixes or Prompt.ask("Hit enter to show the commands")
                 console.print(commands)
 
-                Prompt.ask("Hit enter to continue")
-                continue_ = Prompt.ask(
+                _ = attempt_fixes or Prompt.ask("Hit enter to continue")
+                continue_ = attempt_fixes or Confirm.ask(
                     f"Have you backed up your custom dashboards (if necessary), deleted the {daemonset_name} daemonset and updated the kube-prometheus-stack CRDs?",
-                    choices=["y", "N"],
-                    default="N",
+                    default=False,
                 )
-                if not continue_ == "y":
+                if not continue_:
```

Comment:
No need to use `attempt_fixes` here as it will only be evaluated when `attempt_fixes` is `False` or the user wants to run the commands manually